draft-ietf-scitt-architecture.md

```diff
@@ -165,6 +165,21 @@ Issuers can register their Signed Statements on one or more Transparency Service
 
 # Introduction
 
+## NEW
+This document defines an architecture, message flows, and a base set of extensible message structures to make signed data items transparent via verifiable data structures that are maintained by corresponding transparency services.
+The goal of the transparency enabled by the Supply Chain Integrity, Transparency, and Trust (SCITT) architecture is to enhance auditability and accountability for signed data items (statements) that are about supply chain commodities (artifacts).
+Registering signed statements at a transparency service is akin to a notarization procedure.
```

Comment:
```suggestion
Registering signed statements with a transparency service is akin to a notarization procedure.
```
Feels more natural, but I am not sure, and a review from a native speaker would be ideal :)